cumulusci/cli/cci.py

```diff
@@ -1419,3 +1396,75 @@ def flow_run(runtime, flow_name, org, delete_org, debug, o, skip, no_prompt):
                 "Scratch org deletion failed.  Ignoring the error below to complete the flow:"
             )
             click.echo(str(e))
+
+
+@error.command(name="info", help="Outputs the last part of the most recent traceback")
+@click.argument("num_lines", required=False)
```

Comment:
I definitely don't think this should be an argument, but rather an option (`-n`/`--num_lines`). Inconsistency in how we're using these across commands means they can be confusing for folks (for example, see `cci org scratch`). 

Also, I'd expect us to expand this command in the future to connect to ErrorsDB.